WindowsMSys2_InstallGuide.md

```diff
@@ -1,88 +1,76 @@
 ## Compilation on Windows
 
-### Qt Installation
+The preferred build method on Windows is by using MSYS2 which is a collection
+of packages providing unix-like tools to build native Windows software.
 
-Install Qt via: [Qt Download](http://www.qt.io/download/)  
+Requirements: about 12 GB of free space
 
-Use default options.  
-Add to the PATH environment variable
-
-       ;C:\Qt\5.5\mingw492_32\bin;C:\Qt\Tools\mingw492_32\bin;C:\Qt\Tools\mingw492_32\opt\bin  
-
-Depends on wich version of Qt you use.  
-Change build-all-mingw32make.bat with these values too if you don't use MSys2.  
+The resulting binary is a 32-bit build of Retroshare which will also work
+fine on a 64-bit system.
 
 ### MSYS2 INSTALLATION
 
-Choose your MSYS2 installer here: [MSYS2](http://msys2.github.io/)
-
-Follow install procedure.  
-Don't forget to sync & Update pacman.  
-
-       pacman --needed -Sy bash pacman pacman-mirrors msys2-runtime  
-
-Restart console  
-
-       pacman -Su  
-
-Install all default programms  
+Download MSYS2 from [MSYS2](http://www.msys2.org/). Get the i686 version
+if you run a 32-bit Windows or the x86_64 if you run a 64-bit Windows.
 
-       pacman -S base-devel git mercurial cvs wget p7zip gcc perl ruby python2  
+Run the installer and install MSYS2.
```

Comment:
New versions of msys2 works only on NTFS pertitions. 
So do not install it on SSD, the installation will fail with some unrelated error.

Reply:
SSD? You surely mean an external USB stick with FAT32 or exFAT? I don't think anybody would use those other than for storing files anyway.

Reply:
My existing msys2 installation didn't work after update because of missing files.
The installer stopped because it couldn't copy some OpenSSL related file.
It failed because my SSD are formatted as exFat, and typically other's SSDs too.

From the official site:
Installation restrictions:
    MSYS2 can't be installed on FAT* partitions.
    Current MSYS2 can't be installed on Windows XP anymore

Reply:
I just check my SSDs filesystem and it is NTFS, then I don't have an idea why it is failed on my SSD, but not on my hard drive.